.gitlab-ci.yml

```diff
@@ -60,6 +60,7 @@ variables:
 
   # Publication
   PUBLISH_DOCKER_CLIENT_IMAGES: "false"
+  AUTOMATIC_RELEASE: "false"
```

Comment:
I would include PUBLISH here. Historically we had "PUBLISH_ARTIFACTS" variable (hence the recent clean-up of this in the Release checklist).

I suggest: `PUBLISH_RELEASE_AUTOMATIC` or `AUTOMATIC_PUBLISH_RELEASE`. Even though is less syntactically correct, I prefer the former for the sake of aligning it to the other variable (ie same prefix).

Reply:
+1

Reply:
Yes, much better!